programs/mmm/src/constants.rs

```diff
@@ -11,4 +11,8 @@ pub const MAX_LP_FEE_BP: u16 = 1000;
 pub const ALLOWLIST_MAX_LEN: usize = 6;
 pub const MIN_SOL_ESCROW_BALANCE_BP: u16 = 100;
 
-pub const CANCEL_AUTHORITY: Pubkey = pubkey!("CNTuB1JiQD8Xh5SoRcEmF61yivN9F7uzdSaGnRex36wi");
+pub const CANCEL_AUTHORITY: Pubkey = if cfg!(feature = "anchor-test") {
```

Comment:
maybe we rename this CANCEL_AUTHORITY to DEFAULT_POOL_CHANGE_AUTHORITY

Reply:
Think it's called `CANCEL_AUTHORITY` because it's a keypair used elsewhere for that purpose. In this contract it's currently only being used to close pools with invalid balance.